docs/dashboards/data-pool-and-connectors.md

```diff
@@ -117,6 +117,36 @@ async function run() {
 run();
 ```
 
+## GoogleSheetsConnector Tutorial
+
+Google Sheets is a spreadsheet application included as part of the free, web-based Google Docs Editors suite offered by Google.
```

Comment:
Replace `included as` with `and`.

Reply:
Hmm, text copied from Wikipedia :-)

Reply:
I see. Wikipedia is fixed. 😄

Reply:
No more excuses :D
![image](https://github.com/user-attachments/assets/50d5e75f-8ebc-4c6f-9f65-222d4806617b)